packages/Panel/stories/Panel.tests.stories.js

```diff
@@ -1,17 +1,72 @@
 import React from "react";
+import { testStoryParameters } from "storybook/assets/storyParameters";
 import { getStoryName } from "storybook/storyTree";
-import Screener from "./examples/Screener";
-import { PanelDefaultSticky, PanelOnAfter, PanelFooterSticky, PanelFocusLockDisabled } from "./examples/Cypress";
+import ExampleStory from "storybook/components/ExampleStory";
+import PanelFocusLockDisabled from "./tests/PanelFocusLockDisabled";
+import PanelFooterSticky from "./tests/PanelFooterSticky";
+import PanelOnAfter from "./tests/PanelOnAfter";
+import PanelDefaultSticky from "./tests/PanelDefaultSticky";
+import Screener from "./tests/Screener";
+import Panel from "../src";
 
 const storyName = getStoryName("Panel");
 
 export default {
   title: `${storyName}/Backyard/Tests`,
-  id: "panel-tests",
```

Comment:
Oh, let's keep this! It defines a permalink so if we shuffle the Navigation and move the `<Panel>` to a different category we won't have to fix all the cypress tests!

https://storybook.js.org/docs/react/configure/sidebar-and-urls#permalinking-to-stories